docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -132,6 +197,33 @@ Next, we will:
 1. Copy the example above and paste it into the custom.yaml file.
 1. Save the file.
 
+#### Enterprise license configuration
+
+If you are using Grafana Enterprise Traces, you will need to configure a license, which may be done in one of two ways.
+
+First, you may specify the license text in the `custom.yaml` values file.
+
+```yaml
+license:
+  contents: |
+    LICENSEGOESHERE
+```
+
+Secondly, you may use a secret that contains the license content.
+
+1. Create the secret
+
+```bash
+kubectl create secret generic tempo-license --from-file=license.jwt
+```
```

Comment:
This code block needs to be indented to it appears inline with the number. Otherwise, the numbering will be reset instead of being continuous. 

Same with the number on line 220.

Reply:
Okay, should be intended now, hows that?